src/AppInstallerRepositoryCore/Public/AppInstallerRepositorySource.h

```diff
@@ -82,15 +83,6 @@ namespace AppInstaller::Repository
         virtual SearchResult Search(const SearchRequest& request) const = 0;
     };
 
-    // Interface extension to ISource for locally installed packages.

```

Comment:
Hey, I'm eventually going to get a chance to use that...

Reply:
I'll add it back.